Packaging.props

```diff
@@ -0,0 +1,38 @@
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <!-- Packaging configuration -->
+    <PreReleaseLabel>rc3</PreReleaseLabel> 
+    <PackageDescriptionFile>$(ProjectDir)pkg/descriptions.json</PackageDescriptionFile>
+    <PackageLicenseFile>$(ProjectDir)pkg/dotnet_library_license.txt</PackageLicenseFile>
+    <PackageThirdPartyNoticesFile>$(ProjectDir)pkg/ThirdPartyNotices.txt</PackageThirdPartyNoticesFile>
+    <RuntimeIdGraphDefinitionFile>$(PackagesDir)Microsoft.NETCore.Platforms/1.0.1-rc3-23725/runtime.json</RuntimeIdGraphDefinitionFile>
```

Comment:
I presume this is another place where we need to update the version number. We need to come up with a better solution than modifying many files to update the release version number.